torch/csrc/jit/script/python_sugared_value.cpp

```diff
@@ -282,6 +282,9 @@ std::shared_ptr<SugaredValue> ModuleValue::attr(
     return std::make_shared<OverloadedMethodValue>(
         self_, py::cast<std::vector<std::string>>(overloads));
   }
+  if (!py::hasattr(py_module_, field.c_str())) {
+    throw ErrorReport(loc) << "attribute '" << field << "' does not exist";
+  }
 
   if (py::object attr = py::getattr(py_module_, field.c_str(), py::none())) {
```

Comment:
nit: the third argument here (default if the attribute is not present) is now dead

Reply:
What do you mean by that? I don't think `hasattr` takes a third argument.

Reply:
I mean on getattr

Reply:
I suppose I should just remove the 3rd argument.